backend/clubs/views.py

```diff
@@ -5078,7 +5103,7 @@ def initiate_checkout(self, request, *args, **kwargs):
             )
             if not context or http_status >= 400:
                 raise ApiException(
-                    reason=f"Received {context} with HTTP status {http_status}",
+                    reason=f"Received {context} with HTTP status {status}",
```

Comment:
Where did `status` come from here?

Reply:
Must have slipped through conflict resolution when merging. Will fix it